### PR TITLE
Document known issue to release notes and upgrade guide for 1.16.0

### PR DESCRIPTION
**Pull request by marcboudreau**
When upgrading a Vault server to version 1.16.0, the Vault UI will no longer be able to display the version because it is now using a different endpoint. To account for this, the **default** policy has been updated to allow access to this endpoint, but the change is not applied to an existing **default** policy. This note serves to inform the users that they need to make a small change to the **default** policy to allow the Vault UI to retrieve the version.

### CODE REVIEW

File: website/content/partials/known-issues/1_16-default-policy-needs-to-be-updated.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# redaction like the unauthenticated endpoints that provide the Vault version.
+path "sys/internal/ui/version" {
+    capabilities = ["read"]
+}
+' > default-policy.hcl
+$ vault policy write default ./default-policy.hcl
+```
+
+This change to the default policy will allow the Vault UI to successfully query
+the Vault server version and display it to the users.
```
Comment (schavis) [inline website/content/partials/known-issues/1_16-default-policy-needs-to-be-updated.mdx:43]:
```suggestion
```

Folded into the previous edit/suggestion

**Review by schavis — CHANGES_REQUESTED** [review]

**Comment by tomhjp** [comment]
@schavis @marcboudreau I also have some documentation I would like to add to the upgrading to 1.16 page but it's going to cause a lot of conflicts if I try to land it before this PR. Please could we either expedite this PR or split out the creation of the page from the specific content getting added to the page?

File: website/content/partials/known-issues/1_16-default-policy-needs-to-be-updated.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+### Default policy needs to be updated in order for the version to appear in the Vault UI
+
+#### Affected versions
+
+- 1.16.0
```
[the remark this answers is not in the document]
Comment (marcboudreau) [reply website/content/partials/known-issues/1_16-default-policy-needs-to-be-updated.mdx:5]: Good point, will fix.

**Comment by marcboudreau** [comment]
@tomhjp I'm going to try and get this merged ASAP.

**Review by schavis — APPROVED** [review]

**Pull request merged**